tests/openqa/install/openqa_bootstrap.pm

```diff
@@ -24,7 +24,14 @@ sub run {
     if (has_selinux) {
         script_run('setsebool -P httpd_can_network_connect 1');
     }
-    zypper_call('in openQA-bootstrap');
+    if (is_leap) {
+        # https://progress.opensuse.org/issues/180905
+        assert_script_run('mkdir -p /usr/share/openqa/script/');
+        assert_script_run('curl -L https://raw.githubusercontent.com/os-autoinst/openQA/refs/heads/master/script/openqa-bootstrap > /usr/share/openqa/script/openqa-bootstrap');
+        assert_script_run('test -e /usr/share/openqa/script/openqa-bootstrap && chmod +x /usr/share/openqa/script/openqa-bootstrap');
+    }
+    else { zypper_call('in openQA-bootstrap'); }
+
```

Comment:
Don't! The test target is the state of packages within openSUSE/SLE hence we must *not* download "random packages" from the internet

Reply:
I would say that I was following the suggestions. but I understand your point.